incubator/kafqa/templates/telegraf.configmap.yaml

```diff
@@ -36,6 +40,14 @@ data:
         99
       ]
       service_address = ":8125"
+{{- end }}
+
+{{- if .Values.telegraf.prometheusEnabled }}
+
+    [[inputs.prometheus]]
+      urls = ["http://localhost:{{ .Values.prometheus.port }}/metrics"]
+      namepass = ["kafqa*"]
```

Comment:
Do we still need this ?? Wasn't the metric issue related to telegraf version and not the number of metrics, we can have the namepass configurable, rather than hard coding it here.